docs/user/integrations.rst

```diff
@@ -1,226 +1,19 @@
-VCS Integrations
-================
+Connecting git repositories
```

Comment:
```suggestion
Build docs on every git commit
```

I think this is something closer to what we want? 

I think this page should probably:

* Talk about the benefits of always up to date docs
* I think probably we want to discuss versioning in here, since it relies directly on Git?
* Have a small diagram that shows (You --push--> GitHub --webhook--> RTD --Build docs--> Deploy
  - (Perhaps reuse this: https://about.readthedocs.com/images/homepage.png)
 

But overall I think this is a tricky page to write as currently framed. I think we actually want something like `Benefits of Continuous Docs Deployment` or something?

Reply:
For the title, is "Continuous" the most important word?

"Continuous deployment with git"
"Publish or preview every commit - Continuous deployment with git"
"Git automation"

Closely aligned with this, it would be nice to have an explanation article covering "Documentation as Code" in the lead-up to talking about "Continuous".

Reply:
I'm adding this to a sphinx comment at the top of the article, to make its scope clear.

Reply:
Not 100% sure what the best option is. Our doc refactor had these high-level topics. Should it be one of those? 


<img width="574" alt="Screenshot 2022-11-29 at 11 40 32 AM" src="https://user-images.githubusercontent.com/25510/204631917-ea133956-df0e-4c6d-b509-84e80e9922f0.png">

Reply:
Great idea - it seems best placed as a sublevel immediately under Explanation. These are necessary to click on to get to the subsublevels. So the UX will be consistent, and we won't end up with subsublevels like "Introduction" or "Overview", we can use the sublevel for that.

Reply:
I've added a comment about this in the planning document. We'll have to make some assessment about the sub-levels at that point.